example_workspace/src/image_transport_sample_app/jni/src/main/cpp/image_transport_plugin_test.cpp

```diff
@@ -50,14 +50,7 @@
 #define  LOGI(...)  __android_log_print(ANDROID_LOG_INFO, LOG_TAG, __VA_ARGS__)
 #define  LOGE(...)  __android_log_print(ANDROID_LOG_ERROR, LOG_TAG, __VA_ARGS__)
 
-int main();
-
-void android_main(struct android_app* state) {
-  app_dummy(); // needed so the Android glue does not get stripped off
```

Comment:
No need of `app_dummy`?

Reply:
App dummy is deprecated. This lines have to be added to the CMakeLists.txt https://github.com/Intermodalics/ros_android/blob/20b4e3292e69399e457d5fa105f857c8392a2ec1/example_workspace/src/image_transport_sample_app/jni/CMakeLists.txt#L32-L35 in order to be sure ANativeActivity_onCreate is exported.